modules/convert/user.go

```diff
@@ -55,6 +55,8 @@ func toUser(user *models.User, signed, authed bool) *api.User {
 		result.IsAdmin = user.IsAdmin
 		result.LastLogin = user.LastLoginUnix.AsTime()
 		result.Language = user.Language
+		result.IsActive = &user.IsActive
```

Comment:
only admin & user itselve should know this info?

Reply:
I thought that may be a sensitive information.

Reply:
Oh sorry you already do, didnt expand the diff